packages/itwinui-react/src/core/Buttons/Button.tsx

```diff
@@ -108,6 +117,14 @@ export const Button = React.forwardRef((props, ref) => {
           {endIcon}
         </Box>
       )}
+
+      {loading && (
+        <ProgressRadial
+          size='small'
+          className='iui-button-spinner'
+          aria-hidden
```

Comment:
> * This PR currently just disables the button, which is not enough to make it accessible. I was playing around with live regions to announce the "Loading" text, but that will require some larger changes, which can come in a follow-up PR.

Would removing `aria-hidden` and relying on the visually hidden "Loading." text in the ProgressRadial not be sufficient?

I guess a using a live region will be able to announce when the loading is complete. Because with the regular visually hidden text, when the loading is complete, VoiceOver just reannounces the button label instead of saying something like "finished loading".

Reply:
We cannot rely on text inside the button to be announced at the right time. 

The expected flow should go something like:
1. "click me, button")
2. (user clicks)
3. "processing…" (notice the lack of "click me")
4. (wait N seconds)
5. "done!"